styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```

Comment:
- [ ] Great job here 💯 But according to the requirements you should use a class to select the h1 element, so Kindly add a class to the h1 tag and use it to change the color in your CSS file 👍

![image](https://user-images.githubusercontent.com/45231276/198861301-d051b920-0da0-43e7-9687-bc59e7d172f9.png)